ArduCopter/sensors.cpp

```diff
@@ -197,7 +197,7 @@ void Copter::read_battery(void)
 
     // check for low voltage or current if the low voltage check hasn't already been triggered
     // we only check when we're not powered by USB to avoid false alarms during bench tests
-    if (!ap.usb_connected && !failsafe.battery && battery.exhausted(g.fs_batt_voltage, g.fs_batt_mah)) {
+    if (!ap.usb_connected && !failsafe.battery && battery.exhausted(g.fs_batt_voltage, rtl_mah_calc())) {
```

Comment:
I just thought calling rtl_mah_calc() at 10 Hz is way too frequently.
There are a huge of calculations, will that affect overall FC performance?